__tests__/app.test.js

```diff
@@ -163,6 +164,113 @@ describe('GET /api/articles/:article_id/comments', () => {
     })
 })
 
+describe('POST /api/articles/:article_id/comments', () => {
+    test('201: adds your comment for the specified article ', () => {
+        const newComment = {
+            username: 'rogersop',
+            body: 'tldr'
+        };
+        return request(app)
+        .post('/api/articles/2/comments')
+        .send(newComment)
+        .expect(201)
+        .then(({body}) => {
+            const myComment = body.comment
+                expect(myComment).toMatchObject({
```

Comment:
I like the explicit assertion here to be sure the author and body match what was posted to the endpoint